CONTRIBUTING.md

```diff
@@ -0,0 +1,102 @@
+# Contributing guidelines
+
+# Reporting broken sites
+
+Report broken websites using the "Report Broken Site" link on the extension popup.
+
+# Reporting bugs
+
+- First check to see if the bug has not already been [reported](https://github.com/duckduckgo/duckduckgo-privacy-extension/issues).
+- Make sure that you are using the latest version of the extension.
+
+**How to write a bug report**
```

Comment:
IMO this section is redundant now that we have a bug reporting template. We can link directly to that template from here like this: 

https://github.com/brainly/html-sketchapp/issues/new?template=html-is-not-imported-correctly-to-sketch.md